django_mongodb/features.py

```diff
@@ -60,6 +61,23 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         # Length of null considered zero rather than null.
         "db_functions.text.test_length.LengthTests.test_basic",
     }
+    # $bitAnd, #bitOr, and $bitXor are new in MongoDB 6.3.
+    django_test_expected_failures_bitwise = {
+        "expressions.tests.ExpressionOperatorTests.test_lefthand_bitwise_and",
+        "expressions.tests.ExpressionOperatorTests.test_lefthand_bitwise_or",
+        "expressions.tests.ExpressionOperatorTests.test_lefthand_bitwise_xor",
+        "expressions.tests.ExpressionOperatorTests.test_lefthand_bitwise_xor_null",
+        "expressions.tests.ExpressionOperatorTests.test_lefthand_bitwise_xor_right_null",
+        "expressions.tests.ExpressionOperatorTests.test_lefthand_transformed_field_bitwise_or",
+    }
+
+    @cached_property
+    def django_test_expected_failures(self):
+        expected_failures = super().django_test_expected_failures
+        expected_failures.update(self.django_test_expected_failures_)
```

Comment:
I think the local expected failures should be here or with a _ at the beginning. Is this attribute used somewhere else?

Reply:
I made it an attribute so that it wouldn't have to be excessively indented within this method (and al those lines changed in a commit). It looked like:
```
expected_failures.update(
    {
        ...
    }
)
```
And we can rename easily (without more identation changes) it if we don't need the `@cached_property` anymore.

I can move the underscore to the beginning if you think that's better. I thought it looked more consistent with `django_test_expected_failures_bitwise` to put the underscore at the end, but I suppose that one could use an underscore prefix too.